app/controllers/admins.js

```diff
@@ -0,0 +1,15 @@
+const logger = require('../logger');
+const adminsService = require('../services/admins');
+
+module.exports.signUp = (req, res, next) => {
+  adminsService
+    .signUp(req.body)
+    .then(() => {
+      logger.info(`Created admin user: ${JSON.stringify(req.body.firstName)}`);
```

Comment:
According to our [backend standards document](https://github.com/Wolox/tech-guides/blob/master/backend/backend-standard-and-best-practices.md#21--input-and-output-parameters-naming), request parameters should be in snake_case.